spec/lib/locations/reader_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'fakeredis'
+
+RSpec.describe Locations::Reader, '#call' do
+  let(:now) { Time.zone.local(2015, 6, 30, 14) }
+  let(:locations_ttl) { 1 }
+  let(:key) { 'locations' }
+  let(:json) { 'json' }
+  let(:path) { '/path/to/locations.json' }
+  let(:redis) { Redis.new }
+  let(:reader) { described_class.new(path, redis) }
+
+  def json_in_redis
+    redis.get(key)
+  end
+
+  def keys_in_redis
+    redis.keys('*')
+  end
+
+  def expiry_in_seconds
+    redis.ttl(key)
+  end
+
+  subject(:read) { reader.call }
+
+  before do
+    Timecop.freeze(now)
+    redis.flushdb
+    ENV['LOCATIONS_TTL'] = locations_ttl.to_s
```

Comment:
Agreed. Perhaps passing it in on the constructor (with a default set off the ENV) would be cleaner?